src/main/java/SpringAIController.java

```diff
@@ -0,0 +1,45 @@
+import com.openai.services.AIServiceHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author madhankumar
+ */
+@RestController
+@RequestMapping("/api/v1")
+public class SpringAIController {
+
+	@Autowired
+	AIServiceHelper aiService;
+
+	/**
+	 * Handles a question-answer request by providing a response generated by the AI service.
+	 *
+	 * @param question The question asked by the user.
+	 * @return The response generated by the AI service.
+	 */
+	@GetMapping("/qa")
+	public String chat(@RequestParam String question) {
+		return aiService.chat(question);
+	}
+
+	/**
+	 * Generates a document based on the provided topic using the AI service.
+	 *
```

Comment:
Validate input parameter 'question' to prevent potential security vulnerabilities like SQL injection.